pkg/kubelet/cm/devicemanager/plugin/v1beta1/stub.go

```diff
@@ -53,9 +53,13 @@ type Stub struct {
 	// getPreferredAllocFunc is used for handling getPreferredAllocation request
 	getPreferredAllocFunc stubGetPreferredAllocFunc
 
+	// registerControlFunc is used for controlling auto-registration of requests
+	registerControlFunc stubRegisterControlFunc
+
 	registrationStatus chan watcherapi.RegistrationStatus // for testing
 	endpoint           string                             // for testing
 
+	stopWatcher chan bool
```

Comment:
Can you use a chan struct{} and close it instead of pushing `true` to it? That is standard practice for "signal" style channels

Reply:
Sure, will update.

Reply:
This was no longer needed when an fsnotify watcher was added in the stub.